src/Documentation.php

```diff
@@ -38,31 +47,32 @@ public function __construct(Yaml\Parser $yamlParser, PageBuilder $pageBuilder, $
         $this->load();
     }
 
-    public function load()
+    public function load(): void
     {
         $dirs = (new Finder())->directories()->in($this->versionDir)->sortByName()->depth(0);
 
         foreach ($dirs as $dir) {
-            /** @var SplFileInfo $dir */
+            /* @var SplFileInfo $dir */
```

Comment:
Derp

Reply:
That's something coming from the CS fixer … NFI what to do about it yet, meant to bring it up with you in Slack … but away from home ATM.

Reply:
🤦‍♂️ Yeah I remember that now. It might be a comment fixer to tweak